### PR TITLE
Workflow to auto deploy and test linked token contracts

### PR DESCRIPTION
**Pull request by mb1896**
Workflow link: https://github.com/consensus-shipyard/ipc/actions/workflows/deploy-linked-token-to-dedicated-host.yaml

Closes ENG-750

### CODE REVIEW

File: .github/workflows/deploy-linked-token-to-dedicated-host.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: Deploy LinkedToken contracts to dedicated host running IPC subnet
+
+on:
+  workflow_dispatch:
+  pull_request:
```
Comment (mb1896) [inline .github/workflows/deploy-linked-token-to-dedicated-host.yaml:5]: Will remove `pull_request:` before PR merges.

**Comment by jsoares** [comment]
Closes ENG-750

I don't think this works in a comment, but I'll test anyway :)

**Comment by raulk** [comment]
The LinkedToken is now an upgradable contract, so we'll have to check if these deployment scripts require change.

**Comment by snissn** [comment]
@raulk I have been working on changes here to support the upgrade semantics - https://github.com/consensus-shipyard/ipc/pull/854 the process is a bit frustrating because of the slow timeline to run each test and I think the set up is flaky if multiple tests run concurrently

**Pull request closed without merging**